features/javascript-basics.yml

```diff
@@ -0,0 +1,103 @@
+name: JavaScript basics
```

Comment:
I usually have a bit of an allergic reaction to the word "basic" (same with "simple", "obvious", "minor", etc.) because what may feel basic to one person may not feel the same for someone else.

Looking at the BCD keys, this is mostly built-in objects, statements, and operators. How about these proposals:

* JavaScript built-ins, statements, and operators
* JavaScript language foundation
* JavaScript initial features

I guess it's hard because this feature doesn't represent a well defined feature, but instead a group of features that are old enough to not really matter as separate things. I appreciate that my first proposal may be problematic because there are likely other "statements", or "built-in" things that we want to have as separate features.

In that sense, my last proposal might fit better. So maybe:

* JavaScript (initial support)

Reply:
Thanks Patrick and Dietrich! I like the word "core" a lot more than "basics" so I switched to that and also changed the description to take into account some of Patrick's thoughts.